openlibrary/plugins/openlibrary/js/merge-request-table/index.js

```diff
@@ -277,3 +277,96 @@ function toggleMergeLink(mergeLink) {
         mergeLink.classList.toggle('hidden')
     }
 }
+
+/**
+ * Toggle a dropdown menu while closing other dropdown menus.
+ *
+ * @param {Event} event
+ * @param {string} menuButtonId
+ */
+function toggleAMenuWhileClosingOthers(event, menuButtonId) {
+    // prevent closing of menu on bubbling unless click menuButton itself
+    if (event.target.id === menuButtonId) {
+        // close other open menus, then toggle selected menu
+        closeOtherMenus(menuButtonId)
```

Comment:
```suggestion
        closeOtherMenus(menuButtonId)
```
You can just add `hidden` to each `.dropdown-header` here, and remove the `closeOtherMenus` function.

Reply:
Removing that function will result in the dropdowns overlaying on each other (like shown below) if the drop down menu buttons are clicked one after another. Just wanted to double-check if that's what is desired from a UI standpoint.

![image](https://github.com/internetarchive/openlibrary/assets/52016758/0c262a9e-4a81-47cb-ab60-5d35079526ef)

Reply:
This is generally what I'm asking for: https://github.com/internetarchive/openlibrary/pull/7868/files#r1202903236

Reply:
Make sure that `dropMenus` is a file-level variable that is populated during `initFilters()`.  This way, we won't have to query for elements every time a patron interacts with the header.

Reply:
I created file level variables for the pertinent dropdown menu elements. For the closing of dropdown menus, I tried the code you suggested but (and maybe I'm misunderstanding something) I believe it results in two bugs (1. clicking on the dropdown menu button does not un-toggle the dropdown menu, 2. clicking on the input element of the dropdown menu, i.e. trying to filter reviewers by string, will close the dropdown menu) so I left it as is.

Reply:
The event bubbling is causing number 2.